components/suggest/src/store.rs

```diff
@@ -2313,4 +2314,14 @@ mod tests {
 
         Ok(())
     }
+
+    #[test]
+    fn unparsable_record_serialized_correctly() -> anyhow::Result<()> {
+        let unparseable_record = UnparsableRecord { schema_version: 1 };
+        assert_eq!(
+            serde_json::to_string(&unparseable_record).unwrap(),
+            "{\"v\":1}".to_string()
+        );
```

Comment:
```suggestion
        assert_eq!(
            serde_json::to_value(&unparseable_record)?,
            json!({ "v": 1 }),
        );
```

Just a small style suggestion—I have a slight preference for `json!` over serialized strings (but it's totally a judgment call; please feel free to keep `to_string()` if you like it more), and, since this function returns an `anyhow::Result`, we can use `?`.

Reply:
> I have a slight preference for json! over serialized strings

I like this more too! I'll pull this in.